docs/torch.md

```diff
@@ -1 +1,40 @@
 # PyTorch bindings
+
```

Comment:
Warning: dumb question 😊

Why are we keeping two identical files for the pytorch bindings?

Reply:
I also commented on that, We should keep the torch.md and remove the torch/README.md

We should also thin out the main README.md and make it link to the documentation, just include the most important pieces (installation, authentication, upload / download).

Reply:
Yeah, makes total sense. I'll remove the one in `torch`.